schemas/polybase.ts

```diff
@@ -82,10 +86,38 @@ collection AIModelCommits {
   constructor (owner: User, id: string, commitLog: string, version: Version, fineTuning: FineTuningCommits[]) {
     this.id = id;
     this.name = name;
+    this.owner = owner;
     this.commitLog = commitLog;
     this.version = version;
     this.fineTuning = fineTuning;
   }
 }
-`;
+`
+
+async function load(privateKey: string, polybaseNamespace: string) {
```

Comment:
It seems like we should call this from a context, so that we have access to the database and can use the signer from auth.

Reply:
Hmm. Yeah, I guess we could. This is supposed to be like an admin operation to run migrations outside of the normal flow. I set it up as the mom run migrate so things exist, then we shouldn't need it again

Reply:
I'm still a little unclear about how the web3 DB stuff works and where you manage things.. was thinking this is like deploy pipeline and then the built app just relies on it